agent/04_agent_configure.sh

```diff
@@ -461,7 +461,10 @@ EOF
 function oc_mirror_mce() {
    tmpimageset=$(mktemp --tmpdir "mceimageset--XXXXXXXXXX")
    _tmpfiles="$_tmpfiles $tmpimageset"
-
+   # TODO: Remove this once LSO is published to the 4.12 catalog.
+   if [[ ${OPENSHIFT_RELEASE_STREAM} == "4.12" || ${OPENSHIFT_RELEASE_STREAM} == "4.13" ]]; then
```

Comment:
Need to add another conditional so this is only changed when using MCE. Also it would be useful to log a warning message so its obvious that the release stream being used is not the one the user set
if [ ! -z "${AGENT_DEPLOY_MCE}" ]; then

Reply:
Condition added!